lib/bridge/github/issues.rb

```diff
@@ -16,9 +16,9 @@ def issues(label = nil, opts={})
 
       issues_response = gh.issues(params) do |request|
         request.headers["Accept"] = "application/vnd.github.squirrel-girl-preview"
-      end
+      end.all
 
-      issues_response.all.each{
+      issues_response.each{
```

Comment:
`gh.issues(params)` returns a `Ghee::Proxy`, `all` initializes the call, for some reason assigning the proxy to `issue_response` and then calling `all` on a subsequent invocation seems to make the proxy double up on the request.